src/main/java/org/opentestsystem/ap/ims/util/DataLoader.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2017 Regents of the University of California.
+ *
+ *  Licensed under the Educational Community License, Version 2.0 (the "license");
+ *  you may not use this file except in compliance with the License. You may
+ *  obtain a copy of the license at
+ *
+ *  https://opensource.org/licenses/ECL-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.opentestsystem.ap.ims.util;
+
+import org.opentestsystem.ap.ims.entity.WorkflowStatus;
+import org.opentestsystem.ap.ims.repository.WorkflowStatusRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConditionalOnProperty(value = "data.loader.enabled", havingValue = "true", matchIfMissing = false)
+public class DataLoader implements ApplicationRunner {
+
+    private final WorkflowStatusRepository workflowStatusRepository;
+
+    @Autowired
+    public DataLoader(final WorkflowStatusRepository workflowStatusRepository) {
+        this.workflowStatusRepository = workflowStatusRepository;
+    }
+
+    @Override
+    public void run(ApplicationArguments applicationArguments) throws Exception {
```

Comment:
I am getting to the migration now, it's my last story for the sprint, the flyway one, so technically that above is not the migration.  That is strictly for running locally.  If you look in the root of the project at application.yml there is a profile called 'local' which expects you to have a local MySQL.  JPA is configured to create-drop the database then the DataLoader is called.   When running IMS without the 'local' profile, so by default, it will run against the AWS DEV db instance which already has the table and data loaded.

I'll look at flyway but Jeff J, Todd, Mark, and I discussed Flyway and most are only using it for local development, and possibly a shared DB in DEV.  

All of them were in agreement that they would never run Flyway in Production automatically as it is too risky.  And they would never have a devops person run the script via Maven or Gradle.

TDS creates a deployment ZIP with all the config repo files, migration scripts, and instruction for someone at TDS to follow and do the deployment.  They expect that person to execute the appropriate migration scripts.   

I start to wonder what value Flyway has if it is just for development.  Could we get by without Flyway and only have migration scripts written in plain SQL?